### PR TITLE
feat: support mobile focus, edit(#593)

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

#593 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/view/container.tsx [hunk 1]
```diff
@@ -36,11 +37,80 @@ interface StoreProps {
...
+    pageX: number;
+    pageY: number;
+    timestamp: number;
+  };
+}
+
 type Props = OwnProps & StoreProps & DispatchProps;
 
+const DOUBLE_TAP_DURATION = 200;
+const TAP_THRESHOLD = 10;
```
Comment (js87zz) [inline src/view/container.tsx]: 지금 보니 추후 패키지 구조 정리 작업 때 컴포넌트 내부의 상수랑 타입정의는 별도 파일로 분리하는게 좋겠군요. 🤔

File: src/view/container.tsx [hunk 2]
```diff
@@ -36,11 +37,80 @@ interface StoreProps {
...
+      timestamp: 0
+    }
+  };
+
+  private handleTouchStart = () => {
+    if (!this.el || !isMobile()) {
+      return;
+    }
+
+    this.touchEvent.start = true;
```
Comment (js87zz) [inline src/view/container.tsx]: 마우스 이벤트랑 중첩되지 않도록 터치이벤트에는 `preventDefault()`를 넣어주는게 좋을 것 같습니다. 
https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Supporting_both_TouchEvent_and_MouseEvent

File: src/view/container.tsx (hunk 1 above)
Comment (jung-han) [reply src/view/container.tsx]: 지금 하나 파일 분리해놓을까요?

Comment (jung-han) [reply src/view/container.tsx]: 이후에 분리하도록 하겠습니다!

**Pull request merged**